toolkit/components/uniffi-bindgen-gecko-js/fixtures/tests/test_arithmetic.js

```diff
@@ -7,13 +7,20 @@ add_task(async function() {
     Assert.ok(Arithmetic.IntegerOverflow)
     Assert.equal(await Arithmetic.add(2, 4), 6)
     Assert.equal(await Arithmetic.add(4, 8), 12)
-    // This doesn't work because JS number values are actually floats, and
-    // large integers get rounded to a different value.
-    //
+    // For other backends we would have this test:
     // await Assert.rejects(
     //     Arithmetic.add(18446744073709551615, 1),
     //     Arithmetic.IntegerOverflow,
     //     "add() should throw IntegerOverflow")
+    //
+    // However, this doesn't work because JS number values are actually 64-bit
```

Comment:
This confused me for a while - I thought we were still in a generated directory - but we are actually *under* a directory that has just generated files?